crm_claim_rma/wizards/account_invoice_refund.py

```diff
@@ -39,6 +39,6 @@ def _default_description(self):
     def compute_refund(self, mode='refund'):
         self.ensure_one()
         invoice_ids = self.env.context.get('invoice_ids', [])
-        if invoice_ids:
+        if invoice_ids and invoice_ids[0]:
```

Comment:
better wrap the line above this one in `filter(None, ...)`

But this looks a lot to my like this is fixing symptoms. Could you determine where the falsy id comes from?

Reply:
The problem here is that if the invoice_id on a claim is None then the whole refunding process will not work. You need to provide the invoice of which you want to make a refund if you want to use the standard mechanism, otherwise just create an invoice with the appropriate type `out_refund`. So would it be better if I made the invoice_id field required?

https://github.com/OCA/rma/blob/8.0/crm_claim_rma/views/crm_claim.xml#L87

Reply:
how about hiding the button if the field is False?

Reply:
Sure, hidden